src/span.js

```diff
@@ -18,6 +17,25 @@ export default class Span {
     // ---------------------------------------------------------------------- //
 
     /**
+     * Returns the SpanContext object associated with this Span.
+     *
+     * @return {SpanContext}
+     */
+    context() {
+        if (API_CONFORMANCE_CHECKS) {
+            if (arguments.length !== 0) {
+                throw new Error('Invalid number of arguments');
+            }
+        }
+        let spanContextImp = null;
+        if (this._imp) {
+            spanContextImp = this._imp.context();
+        }
+        // If there's no imp...
```

Comment:
nit: this comment isn't accurate as line 35 is run for both branches (though I realize this comment may have been copied & pasted from somewhere else).